hasura/schema/admin/schema.graphql

```diff
@@ -35257,12 +35274,14 @@ Ordering options when selecting data from "profiles".
 """
 input profiles_order_by {
   address: order_by
+  address_data_fetches: address_data_fetches_order_by
   app_emails: order_by
   avatar: order_by
   background: order_by
   bio: order_by
   chat_id: order_by
   claims_aggregate: claims_aggregate_order_by
+  colinks_notification_emails: order_by
```

Comment:
I prefer to do yarn docker:stop and yarn docker:start and resync metadata so that the generated files will reflect only my changes

Reply:
removed. thanks